### PR TITLE
Th add sprites

### PR DESCRIPTION
**Pull request by tylerjhevia**
@bretthev

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,13 +6,29 @@
...
+      Press 'Space' or click the mouse to flap. </br>
+      Avoid the obstacles!
+    </p>
 
-  <h3>Score: <span class='score'>0</span></h3>
+    <canvas id='game' width='600' height='600'>
+      Flappy Bird
+    </canvas>
 
+    <div class='difficulty-div'>
```
Comment (bretthev) [inline index.html:20]: Drop the '-div.'

File: lib/game.css [hunk 2]
```diff
@@ -6,14 +16,35 @@
...
-  font-size: 50px;
+  color: white;
+  font-size: 30px;
   margin: auto;
   text-align: center;
 }
 
 .score {
-  font-size: 50px;
+  font-size: 30px;
```
Comment (bretthev) [inline lib/game.css:47]: You're repeating font-sizes here, can you abstract this out to a container element to DRY it up?

File: lib/game.js [hunk 3]
```diff
@@ -5,8 +5,10 @@ var Obstacle = require('./Obstacle.js')
 const canvas = document.getElementById('game')
 const context = canvas.getContext('2d')
 const score = document.querySelector('.score')
-let currentScore = 0;
-const bird = new Bird(101, 75, 20, 20, 'green')
+var currentScore = 0;
```
Comment (bretthev) [inline lib/game.js:8]: Why the change away from es6 here? Just use `const` or `let`

File: lib/game.js [hunk 4]
```diff
@@ -30,9 +32,15 @@ let groundDirection = {
...
 }
 
+context.font = "40px Arial";
+context.fillStyle = "white";
+context.fillText("Press ENTER to play", 175, 300, 300);
+
 function saveHighScore() {
-  if (currentScore > parseInt(localStorage.getItem('highScore'))) {
-  localStorage.setItem('highScore', currentScore);
+  var endingScore = parseInt(score.innerHTML, 10)
```
Comment (bretthev) [inline lib/game.js]: change to es6

File: lib/game.js [hunk 5]
```diff
@@ -30,9 +32,15 @@ let groundDirection = {
...
+  var highScore = (localStorage.getItem('highScore'), 10)
+  if (endingScore > highScore) {
+  localStorage.setItem('highScore', endingScore)
```
Comment (bretthev) [inline lib/game.js]: indentation

File: lib/game.js [hunk 6]
```diff
@@ -68,43 +76,47 @@ function drawGround () {
...
 }
 
+function drawBird (direction) {
+  bird.draw(context)
+  bird.fly(direction)
+}
+
 function checkForCollision () {
   obstacles.forEach(function(obstacle) {
     if (bird.x < obstacle.x + obstacle.width &&
```
Comment (bretthev) [inline lib/game.js:90]: could you maybe write a function for finding the corners of an obstacle/the bird? just to avoid doing all this adding and checking for corners manually

File: lib/game.js [hunk 7]
```diff
@@ -68,43 +76,47 @@ function drawGround () {
...
   drawObstacles();
+  drawGround();
   checkForCollision()
   updateScore();
-  saveHighScore();
   requestAnimationFrame(gameLoop)
 }
 
 window.addEventListener('keyup', function (event) {
-  if (event.keyCode === 32) { bird.flap() }
```
Comment (bretthev) [inline lib/game.js:99]: stylistic/opinion thing here, but I generally prefer to inline super short if statements

**Review by bretthev — CHANGES_REQUESTED** [review]
I didn't actually pull this down and try and play or anything, but I've got some suggestions. Gonna look at some more of the game logic now.

**Comment by bretthev** [comment]
Also, does the `th` in the branch name stand for "Tyler Hevia?" Because that seems like a hilariously unnecessary naming convention. Just name your branches what they are. `bug/fixes collision logic` or `refactor/extracts corner logic to own function` or whatever

File: index.html (hunk 1 above)
Comment (tylerjhevia) [reply index.html:20]: why? We're using it to position stuff

Comment (bretthev) [reply index.html:20]: I meant in the class name. So just call it `difficulty` or `difficulty-container` if you have to. The `-div` is unnecessary.
